### ISSUE TITLE
New plugins are placed unpredictably on the patchbay canvas

### ISSUE THREAD

**Issue by unfa** [falkTX/Carla#391]
In bigger projects I tend to spend a lot of time looking for a place where Carla placed node for a new plugin I just added. I can never predict where they will appear.

I'd prefer Carla to place new nodes in the middle of the currently view region of the canvas. This would make my work in Carla much faster.

**Comment by falkTX** [comment]
the nodes are added on their last used location.
I need to add an option for this.

**Comment by mxmilkb** [comment]
I am finding I am spending 15/20% of my time in Carla zooming out and playing hunt-what-I-just-added.

If there's one change that would make the most improvement in the experience of Carla, it would be to have new plugins appear in the middle of the canvas.

**Comment by falkTX** [comment]
This has been implemented in the past few weeks.
New plugins are always placed in the middile of the canvas, or a bit below if the exact center is already occupied by something.

The new canvas stuff, if running under jack multi-client, requires a jack version which supports meta-data.
It is known that pretty much all stock Debian and Ubuntu versions do not have this :(
But I got to push for it anyway, sorry
